schemas/contacts-schemas.js

```diff
@@ -5,13 +5,21 @@ const contactsAddSchema = Joi.object({
         "any.required": "missing required name field",
     }),
     email: Joi.string().required().messages({
-        "any.required": "missing required name field",
+        "any.required": "missing required email field",
```

Comment:
email and phone should be validated properly

Reply:
fixed